exercises/collatz-conjecture/collatz_conjecture.py

```diff
@@ -1,2 +1,2 @@
-def collatz_steps():
+def collatz_steps(n):
```

Comment:
I like `number` better than `n` because it is too ambiguous and we as programmers should avoid such names,
Do you agree?

Reply:
Yeah I suppose it is more clear, and regarding my previous stuff about it matching the readme's, I think in this case n == number is pretty clear